online_store/gt_sport_v1.html

```diff
@@ -0,0 +1,120 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
```

Comment:
There are indentation problems in you HTML and JavaScript. I would suggest two things to clean it up. 

1. use a separate js file for all your javascript
1. run vs codes auto formatted on both this html and the newly created js file. 

I created a video explaining how to use VS Code auto format [here](https://www.youtube.com/watch?v=QeGp3wKU3Jg) with this exact code as an example.

Reply:
The video is amazing and fixed my indentation problems, thanks!
I have now also a separate .js file. Great suggestiongs, thanks.